package.json

```diff
@@ -12,10 +12,10 @@
   },
   "dependencies": {
     "ejs": "^2.5.6",
-    "express": "^4.15.2"
+    "express": "^4.15.2",
+    "got": "^11.1.4"
```

Comment:
This should go into `devDependencies` where `request` was. We can do that by running `npm i -D (or --save-dev) got`